src/Config/ObjectTypeDefinition.php

```diff
@@ -67,6 +73,29 @@ private function treatFieldsDefaultAccess(ArrayNodeDefinition $node): void
         ->end();
     }
 
+    /**
+     * set empty fields.accessConfig with fieldsDefaultAccessConfig values if is set?
+     */
+    private function treatFieldsDefaultAccessConfig(ArrayNodeDefinition $node): void
+    {
+        $node->validate()
+            ->ifTrue(function ($v) {
+                return array_key_exists('fieldsDefaultAccessConfig', $v) && null !== $v['fieldsDefaultAccessConfig'];
+            })
+            ->then(function ($v) {
+                foreach ($v['fields'] as &$field) {
+                    if (array_key_exists('accessConfig', $field) && null !== $field['accessConfig']) {
```

Comment:
```suggestion
                    if (isset($field['accessConfig'])) {
```
The `isset` operator on arrays checks, if the key exists and if it's not null